dotCMS/src/main/java/com/dotcms/ai/api/CompletionsAPI.java

```diff
@@ -42,20 +42,20 @@ static CompletionsAPI impl() {
     void summarizeStream(CompletionsForm searcher, OutputStream out);
 
     /**
-     * this method takes a prompt and returns the AI response based upon that prompt
+     * this method takes a prompt in the form of json and returns a json AI response based upon that prompt
      *
-     * @param promptForm
+     * @param promptJSON
      * @return
      */
-    JSONObject raw(CompletionsForm promptForm);
+    JSONObject raw(JSONObject promptJSON);
```

Comment:
why using jsonobject, should use Map instead

Reply:
@jdotcms JSONObject is used all over the place when it comes to AI API.